apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/TraceLabels.cs

```diff
@@ -24,25 +24,25 @@ namespace Google.Cloud.Diagnostics.Common
     /// <summary>
     /// A helper class to handle span labels.
     /// </summary>
-    internal static class TraceLabels
+    public static class TraceLabels
```

Comment:
Ditto these constants. If they really can't ever change, they're fine of course.